probes.d

```diff
@@ -0,0 +1,97 @@
+                                 uint32_t cwnd, size_t inflight);
+    probe quicly_cc_congestion(struct st_quicly_conn_t *conn, int64_t at, uint64_t max_lost_pn, size_t inflight, uint32_t cwnd);
+
+    probe quicly_transport_close_send(struct st_quicly_conn_t *conn, int64_t at, uint64_t error_code, uint64_t frame_type,
+                                      const char *reason_phrase);
+    probe quicly_transport_close_receive(struct st_quicly_conn_t *conn, int64_t at, uint64_t error_code, uint64_t frame_type,
+                                         const char *reason_phrase);
+    probe quicly_application_close_send(struct st_quicly_conn_t *conn, int64_t at, uint64_t error_code, const char *reason_phrase);
+    probe quicly_application_close_receive(struct st_quicly_conn_t *conn, int64_t at, uint64_t error_code,
+                                           const char *reason_phrase);
+
+    probe quicly_stream_send(struct st_quicly_conn_t *conn, int64_t at, struct st_quicly_stream_t *stream, uint64_t offset,
+                             size_t len, int is_fin);
+    probe quicly_stream_receive(struct st_quicly_conn_t *conn, int64_t at, struct st_quicly_stream_t *stream, uint64_t offset,
+                                size_t len);
+    probe quicly_stream_acked(struct st_quicly_conn_t *conn, int64_t at, int64_t stream_id, uint64_t offset, size_t len);
+    probe quicly_stream_lost(struct st_quicly_conn_t *conn, int64_t at, int64_t stream_id, uint64_t offset, size_t len);
+
+    probe quicly_max_data_send(struct st_quicly_conn_t *conn, int64_t at, uint64_t limit);
+    probe quicly_max_data_receive(struct st_quicly_conn_t *conn, int64_t at, uint64_t limit);
+
+    probe quicly_max_streams_send(struct st_quicly_conn_t *conn, int64_t at, uint64_t limit, int is_unidirectional);
+    probe quicly_max_streams_receive(struct st_quicly_conn_t *conn, int64_t at, uint64_t limit, int is_unidirectional);
+
+    probe quicly_max_stream_data_send(struct st_quicly_conn_t *conn, int64_t at, struct st_quicly_stream_t *stream, uint64_t limit);
+    probe quicly_max_stream_data_receive(struct st_quicly_conn_t *conn, int64_t at, int64_t stream_id, uint64_t limit);
+
+    probe quicly_streams_blocked_send(struct st_quicly_conn_t *conn, int64_t at, uint64_t limit, int is_unidirectional);
+    probe quicly_streams_blocked_receive(struct st_quicly_conn_t *conn, int64_t at, uint64_t limit, int is_unidirectional);
+
+    probe quicly_data_blocked_receive(struct st_quicly_conn_t *conn, int64_t at, uint64_t offset);
+
+    probe quicly_stream_data_blocked_receive(struct st_quicly_conn_t *conn, int64_t at, int64_t stream_id, uint64_t limit);
+
+    probe quicly_quictrace_send(struct st_quicly_conn_t *conn, int64_t at, uint64_t pn, size_t len, uint8_t packet_type);
+    probe quicly_quictrace_recv(struct st_quicly_conn_t *conn, int64_t at, uint64_t pn, size_t len, uint8_t enc_level);
```

Comment:
Is `packet_type` of QUICTRACE_SEND and `enc_level` of QUICTRACE_RECV the same thing?

If so, is there a reason to use different names?

PS. Never mind if this is due to quictrace using different names for the same thing, but I am concerned if they might actually mean different things, that we fail to capture in the way we call the probes.

Reply:
Hm. We don't really use enc_level of QUICTRACE_RECV, so it doesn't matter either. I'm not sure why it's there. Feel free to remove it.

Reply:
Removed in b9c9c1f.